File: 준영/LV1/이상한 문자 만들기.py
def solution(s):
    answer = ''
    idx = 0
    for i in s:
        if i == " ":
            answer += " "
            idx = 0
        else:
            if idx % 2 == 0:
                answer += i.upper()
            else:
                answer += i.lower()
            idx += 1
    return answer

print(solution("AAAAAAAAAAA A A A A AAAAA AAA "))
print(solution("aaaaaaaaaaa a a a a aaaaa aaa "))

File: 준영/프로그래머스/LV1/이상한 문자 만들기.js
function solution(s) {
  var answer = ''
  var idx = 0
  for (const i of s) {
    if (i === " ") {
      answer += " "
      idx = 0
    } else {
      if (idx % 2 === 0) {
        answer += i.toUpperCase()
      } else {
        answer += i.toLowerCase()
      }
      idx += 1
    }
  }
  return answer
}